### DISCUSSION TITLE
Adding mortar for enemies

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1551] [Question and answer] [answered]
Hello i'm planning to add mortar to the enemies with this mod https://steamcommunity.com/sharedfiles/filedetails/?id=3027136288

Thing is the enemy currently is the ISIS from Project Opfor, but this mod provides Hell Cannon for CSAT.

Is it possible to do all of this? 

Cheers.

**Comment by Vdauphin** [comment]
Hello,

Project Opfor is outdated please use an up-to-date mod as 3CB Factions

> Is it possible to do all of this?

To do all what? 

Cheers

**Reply by XDSenDX** [reply]
Forcing ISIS group to use the Hell Cannon instead of the CSAT.

Cheers.

**Reply by XDSenDX** [reply, marked as answer]
Anyways i switched to 3CB Factions and appearently they all have their own mortar so it should be good.
